doc-content/shared-kie-docs/src/main/asciidoc/KieServer/Install-section.adoc

```diff
@@ -84,197 +85,377 @@ container. The Web container version of the WAR contains only the REST interface
-
-|**org.kie.executor.interval**
-|integer (default is 0)
-|Number of time units between polls by executor
-|No
-
-|**org.kie.executor.pool.size**
-|integer (default is 1)
-|Number of threads in the pool for async work
-|No
-
-|**org.kie.executor.retry.count**
-|integer (default is 3)
-|Number of retries to handle errors
-|No
-
-|**org.kie.executor.timeunit**
-|TimeUnit (default is "SECONDS")
-|TimeUnit representing interval
-|No
-
-|**org.kie.executor.disabled**
-|boolean (default is "false")
-|Disables executor completely
-|No
-
-|**kie.server.jms.queues.response**
-|string (default is "queue/KIE.SERVER.RESPONSE")
-|JNDI name of response queue for JMS
-|No
-
-|**org.kie.server.controller.connect**
-|long (default is 10000)
-|Waiting time in milliseconds between repeated attempts to connect kie server to
-                {CONTROLLER} when kie server starts up
-|No
-
-|**org.drools.server.filter.classes**
-|boolean (default is "false")
-|If true, accept only classes which are annotated with @org.kie.api.remote.Remotable
-                or @javax.xml.bind.annotation.XmlRootElement as extra JAXB classes
-|No
-
-|
-|
-|
-|
+|Property
+|Values
+|Default
+|Description
+
+|`org.kie.server.persistence.ds`
+|String
+|N/A
+|A data source JNDI name. Set this property when enabling the BPM support.
+
+|`org.kie.server.persistence.tm`
+|String
+|N/A
+|A transaction manager platform for Hibernate properties set. Set this property when enabling the BPM support.
+
+|`org.kie.server.persistence.dialect`
+|String
+|N/A
+|The Hibernate dialect to be used. Set this property when enabling the BPM support.
+
+|`org.kie.server.persistence.schema`
+|String
+|N/A
+|The database schema to be used.
 |===
+endif::[]
 
-[IMPORTANT]
-====
-If you are running both {KIE_SERVER} and {CENTRAL} you must configure KIE Server to use a different Data Source to {CENTRAL} using the  *org.kie.server.persistence.ds* property.
-{CENTRAL} uses a jBPM Executor Service that can conflict with {KIE_SERVER} if they share the same Data Source.
-====
+.System properties for executor properties
+[cols="2,1,1,3", options="header"]
+|===
+|Property
+|Values
+|Default
+|Description
+
+|`org.kie.executor.interval`
+|Integer
+|`0`
+|The time between the moment the {PRODUCT} executor finishes a job and the moment it starts a new one, in a time unit specified in the `org.kie.executor.timeunit` property.
+
+|`org.kie.executor.timeunit`
+| https://docs.oracle.com/javase/8/docs/api/index.html?java/util/concurrent/TimeUnit.html[`java.util.concurrent.TimeUnit`] constant
+|`SECONDS`
+|The time unit in which the `org.kie.executor.interval` property is specified.
+
+|`org.kie.executor.pool.size`
+|Integer
+|`1`
+|The number of threads used by the {PRODUCT} executor.
+
+|`org.kie.executor.retry.count`
+|Integer
+|`3`
+|The number of retries the {PRODUCT} executor attempts on a failed job.
+
+|`org.kie.executor.jms.queue`
+|String
+|`queue/KIE.SERVER.EXECUTOR`
+|Job executor JMS queue for {KIE_SERVER}.
+|===
```

Comment:
These `kie.executor` properties are also for jbpm/rhpam only.

Reply:
Okay. I will remove those properties from RHDM docs.